src/helpers/notifications/block.article.js

```diff
@@ -28,13 +26,19 @@ export default async (operation, articleSlug) => {
       }
     });
 
+    const emailUrl = `${env.APP_URL_FRONTEND}/api/articles/${articleSlug}`;
+    const inApplUrl = `/articles/${articleSlug}`;
```

Comment:
@nkpremices do you mean `inApplUrl` or `inAppUrl`? if it's deliberate, you can leave it. If it isn't deliberate, then it could lead to bugs/issues for someone.